crates/shielder-scheduler-server/src/handlers/get_status.rs

```diff
@@ -23,22 +23,35 @@ pub struct GetStatusResponse {
 }
 
 #[instrument(level = "info", skip_all)]
-pub async fn get_status(
-    State(state): State<Arc<AppState>>,
+pub async fn get_status<Storage: StorageInterface>(
+    State(state): State<Arc<AppState<Storage>>>,
     Path(last_note_index): Path<String>,
 ) -> impl IntoResponse {
     info!(
         "Received get status request for last_note_index: {}",
         last_note_index
     );
 
-    match get_request_by_last_note_index(&state.db_pool, &last_note_index).await {
+    match state
+        .storage
+        .get_request_by_last_note_index(&last_note_index)
+        .await
+    {
         Ok(Some(res)) => {
             info!(
                 "Found request with status: {:?} for last_note_index: {}",
                 res.status, last_note_index
             );
-            (axum::http::StatusCode::OK, Json(res)).into_response()
+            (
+                axum::http::StatusCode::OK,
+                Json(GetStatusResponse {
+                    last_note_index: res.last_note_index.to_string(),
+                    status: res.status.clone(),
+                    created_at: res.created_at,
+                    processed_at: res.processed_at,
```

Comment:
Please add `relay_after` to the response, see merged PR in meanwhile this is in review: https://github.com/Cardinal-Cryptography/blanksquare-monorepo/pull/55/files